tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "baseUrl": ".",
```

Comment:
nit: According to the docs this should no longer be necessary when using paths because it defailts to the location of the tsconfig.json.